### ISSUE TITLE
Add 8chan.moe support

### ISSUE THREAD

**Issue by User1l0** [K1rakishou/Kuroba-Experimental#847]

**Comment by swindlesmccoop** [comment]
Any updates/work done on this yet?

**Comment by K1rakishou** [comment]
No, and the site doesn't even open for me, maybe it's already dead?

**Comment by swindlesmccoop** [comment]
I just opened it and it seems to work fine for me. I don't think they do any IP range blocking either.

**Comment by K1rakishou** [comment]
![image](https://github.com/K1rakishou/Kuroba-Experimental/assets/10332458/bde3eceb-636a-4abb-ba58-0f084d4aeeca)

???

**Comment by swindlesmccoop** [comment]
?????
![Screenshot_20230529-221447_1](https://github.com/K1rakishou/Kuroba-Experimental/assets/65102316/56c57798-4559-457d-aa39-f82a252d5c44)

**Comment by LaggyAsBalls** [comment]
Asking about this too. If it would be impossible or not.

**Comment by K1rakishou** [comment]
Send your PR and I will merge it.

**Comment by LaggyAsBalls** [comment]
I don't code.

**Comment by User1l0** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

No, this website still works till this day. Please add it to kuroba ex
